### PR TITLE
feat: Add exportSingle command

### PR DESCRIPTION
**Pull request by paultranvan**
This is useful to export a single doc  by its id

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -125,6 +125,14 @@ const handleExportCommand = async args => {
   await ach.export(doctypes, filename, last)
 }
 
+const handleExportSingleCommand = async args => {
+  let { doctype, id, filename, url, token, last } = args
+  token = token || autotoken(url, [doctype])
+  const ach = new ACH(token, url, [doctype])
+  await ach.connect()
+  await ach.exportSingle(doctype, id, filename, last)
```
Comment (Ldoppea) [inline cli.js:133]: `last` arg seems not to be used by `exportDocs()`

**Review by Ldoppea — APPROVED** [review]

**Comment by Crash--** [comment]
cc @paultranvan

File: cli.js (hunk 1 above)
Comment (paultranvan) [reply cli.js:133]: Good catch!

**Review by Crash-- — APPROVED** [review]

**Comment by paultranvan** [comment]
Thanks for waking me up @Crash--

**Pull request merged**